packages/office-ui-fabric-react/src/components/Slider/Slider.base.tsx

```diff
@@ -163,7 +200,18 @@ export class SliderBase extends BaseComponent<ISliderProps, ISliderState> implem
       return Math.max(this.props.min, Math.min(this.props.max, value));
     }
   }
-
+  private _getValueLabel(className: string): JSX.Element | null {
+    const { showValue, disabled, valueFormat, label } = this.props;
+    const value = valueFormat ? valueFormat(this.value!) : this.value;
```

Comment:
You could move this line inside the `if` statement below so that you don't do this calculation if it's not needed.

Reply:
done